test/JDBC/expected/remote-proc-exec-vu-verify.out

```diff
@@ -0,0 +1,1379 @@
+PRINT 'Test 101: Nested SELECT-only procedures - should pass';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Outer_CallsSelectOnly;
+GO
+~~START~~
+varchar#!#int
+Outer before call#!#0
+~~END~~
+
+
+-- Test 102: Nested procedure - outer SELECT, inner INSERT (should FAIL)
+-- Expected: Should fail with error about INSERT in nested procedure
+PRINT 'Test 102: Outer calls inner with INSERT - should fail';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Outer_CallsInsert;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Remote procedure contains INSERT statement)~~
+
+
+-- Test 103: Two-level nesting - all SELECT-only (should PASS)
+-- Expected: Should execute both levels successfully
+PRINT 'Test 103: Two-level nesting all SELECT - should pass';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Level1_DeepNesting;
+GO
+~~START~~
+varchar#!#int
+Level 1 calling Level 2#!#1
+~~END~~
+
+
+-- Test 104: Two-level nesting - DML at level 2 (should FAIL)
+-- Expected: Should fail - INSERT found at level 2
+PRINT 'Test 104: Two-level nesting with DML at level 2 - should fail';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Level1_DeepWithDML;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Remote procedure contains INSERT statement)~~
+
+
+-- Test 105: Circular reference detection (A calls B, B calls A)
+-- Expected: Should fail with circular reference error
+PRINT 'Test 105: Circular reference A->B->A - should fail';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_CircularA;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Circular procedure reference detected: bbf_rpe_server.master.dbo.sp_CircularA)~~
+
+
+-- Test 106: Multiple inner procedure calls (all SELECT-only)
+-- Expected: Should pass - calling same inner proc multiple times
+PRINT 'Test 106: Multiple inner procedure calls - should pass';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Outer_MultipleInner;
+GO
+~~START~~
+varchar
+Calling multiple inner procs
+~~END~~
+
+
+-- Test 107: Outer procedure with dynamic SQL (sp_executesql)
+-- Expected: Should fail - dynamic SQL blocked
+PRINT 'Test 107: Outer with dynamic SQL - should fail';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Outer_WithDynamicSQL;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Remote procedure contains EXECUTE(string) statement)~~
+
+
+-- Test 108: Nested procedure with UPDATE in inner (should FAIL)
+-- Expected: Should fail with error about UPDATE in nested procedure
+PRINT 'Test 108: Outer calls inner with UPDATE - should fail';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Outer_CallsUpdate;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Remote procedure contains UPDATE statement)~~
+
+
+-- Test 109: Nested procedure with DELETE in inner (should FAIL)
+-- Expected: Should fail with error about DELETE in nested procedure
+PRINT 'Test 109: Outer calls inner with DELETE - should fail';
+GO
+EXEC bbf_rpe_server.master.dbo.sp_Outer_CallsDelete;
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Remote procedure contains DELETE statement)~~
```

Comment:
Can you add a test case like : 
insert into local table with exec from a remote server, thanks